### PR TITLE
update

### PR DESCRIPTION
**Pull request by AristarkhV**

### CODE REVIEW

File: src/com/company/FootballMatch.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
+package com.company;
+
+public class FootballMatch extends Service {
+
+    public int MatchOutcome;
+
+    public int MatchForecast(int firstRealScore, int secondRealScore, int firstForecastScore, int secondForecastScore) {
+        if ((firstRealScore == firstForecastScore && secondRealScore == secondForecastScore))
```
Comment (Sofasmile) [inline src/com/company/FootballMatch.java]: where is the ternary operator?

File: src/com/company/FootballMatch.java [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+            MatchOutcome = 2;
+        else {
+            if ((firstRealScore >= secondRealScore && firstForecastScore >= secondForecastScore) ||
+                    ((firstRealScore <= secondRealScore && firstForecastScore <= secondForecastScore)))
+                MatchOutcome = 1;
+            else
+                MatchOutcome = 0;
+        }
+        return MatchOutcome;
+
```
Comment (Sofasmile) [inline src/com/company/FootballMatch.java]: remove redundant line

File: src/com/company/Main.java [hunk 3]
```diff
@@ -1,8 +1,59 @@
...
+                System.out.println("Enter the forecast of team 2:");
+                int Forecast2 = newFootballMatch.getInt();
+                System.out.println("The match is completed - " + RealOutcome1 + ":" + RealOutcome2
+                        + ", user forecast - " + Forecast1 + ":" + Forecast2);
+                System.out.println("Match result: " + newFootballMatch.MatchForecast(RealOutcome1, RealOutcome2, Forecast1, Forecast2));
+                System.exit(0);
+            }
+
+            case 2: {
+
```
Comment (Sofasmile) [inline src/com/company/Main.java]: the same

File: src/com/company/Main.java [hunk 4]
```diff
@@ -1,8 +1,59 @@
...
+                System.out.println("Enter the forecast of team 1");
+                int Forecast1 = newFootballMatch.getInt();
+                System.out.println("Enter the forecast of team 2:");
+                int Forecast2 = newFootballMatch.getInt();
+                System.out.println("The match is completed - " + RealOutcome1 + ":" + RealOutcome2
+                        + ", user forecast - " + Forecast1 + ":" + Forecast2);
+                System.out.println("Match result: " + newFootballMatch.MatchForecast(RealOutcome1, RealOutcome2, Forecast1, Forecast2));
+                System.exit(0);
+            }
+
```
Comment (Sofasmile) [inline src/com/company/Main.java]: the same

File: src/com/company/Main.java [hunk 5]
```diff
@@ -1,8 +1,59 @@
...
+            case 2: {
+
+                Rhombus newRhombus = new Rhombus();
+                System.out.println("Enter rhombus size:");
+                newRhombus.getOutput(newRhombus.getInt());
+                System.exit(0);
+            }
+
```
Comment (Sofasmile) [inline src/com/company/Main.java]: the same

File: src/com/company/Power.java [hunk 6]
```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class Power extends Service {
+
+    static int getPower(int Num, int Pow) {
```
Comment (Sofasmile) [inline src/com/company/Power.java]: read about parameter names and change them
use camelCase
https://mate-academy.github.io/style-guides/java/java.html#s5-naming

Comment (Sofasmile) [reply src/com/company/Power.java]: better use full name of the parameters

Comment (Sofasmile) [reply src/com/company/Power.java]: this method will not work if you write getPower(3, -4)

File: src/com/company/Service.java [hunk 7]
```diff
@@ -0,0 +1,20 @@
...
+    Scanner scan = new Scanner(System.in);
+
+    public int getInt() {
+        int number;
+        if (scan.hasNextInt()) {
+            number = scan.nextInt();
+        } else {
+            System.out.println("You made a mistake when entering a number. Try again.");
+            scan.next();
+            number = getInt();//рекурсия
```
Comment (Sofasmile) [inline src/com/company/Service.java:15]: remove the comment, it is unnecessary

**Review by Sofasmile — COMMENTED** [review]
resolve comments

File: src/com/company/FootballMatch.java [hunk 8]
```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class FootballMatch extends Service {
+
+    public int MatchForecast(int firstRealScore, int secondRealScore, int firstForecastScore, int secondForecastScore) {
+        return ((firstRealScore == firstForecastScore && secondRealScore == secondForecastScore)) ? 2 :
+        ((firstRealScore >= secondRealScore && firstForecastScore >= secondForecastScore)
+                || (firstRealScore <= secondRealScore && firstForecastScore <= secondForecastScore)) ? 1 : 0;
+    }
+
```
Comment (Sofasmile) [inline src/com/company/FootballMatch.java:10]: remove redundant line

File: src/com/company/Power.java [hunk 9]
```diff
@@ -0,0 +1,13 @@
+package com.company;
+
+public class Power extends Service {
+
+    static double getPower(int Number, int Power) {
```
Comment (Sofasmile) [inline src/com/company/Power.java:5]: names of the parameters, methods, variables must start from small letter

File: src/com/company/Rhombus.java [hunk 10]
```diff
@@ -0,0 +1,24 @@
...
+        for (int i = 0; i <= NumbAsterisk; i = i + 2) {
+            for (int putSpace = 0; putSpace < i / 2; putSpace++)
+                System.out.print(" ");
+            for (int putAstr = NumbAsterisk - i - 1; putAstr >= 0; putAstr--)
+                System.out.print("*");
+            System.out.println("");
+        }
+    }
+}
+
```
Comment (Sofasmile) [inline src/com/company/Rhombus.java:23]: remove two redundant lines after }

File: src/com/company/Main.java [hunk 11]
```diff
@@ -1,8 +1,55 @@
 package com.company;
 
+import java.util.Scanner;
+
 public class Main {
```
Comment (Sofasmile) [inline src/com/company/Main.java:5]: better use Application name of the class than Main

File: src/com/company/FootballMatch.java [hunk 12]
```diff
@@ -0,0 +1,12 @@
+package com.company;
+
+public class FootballMatch extends Service {
+
+    public int MatchForecast(int firstRealScore, int secondRealScore, int firstForecastScore, int secondForecastScore) {
+        return ((firstRealScore == firstForecastScore && secondRealScore == secondForecastScore)) ? 2 :
+        ((firstRealScore >= secondRealScore && firstForecastScore >= secondForecastScore)
+                || (firstRealScore <= secondRealScore && firstForecastScore <= secondForecastScore)) ? 1 : 0;
```
Comment (Sofasmile) [inline src/com/company/FootballMatch.java:8]: if you write MatchForecast(2, 2, 3, 2) you must get result = 0
this code will work correctly:
return ((firstRealScore == firstForecastScore && secondRealScore == secondForecastScore)) ? 2 :
        ((firstRealScore > secondRealScore && firstForecastScore > secondForecastScore)
                || (firstRealScore < secondRealScore && firstForecastScore < secondForecastScore))
                || (firstRealScore == secondRealScore && firstForecastScore == secondForecastScore)) ? 1 : 0;

**Review by Sofasmile — COMMENTED** [review]
good, but make small fixing

File: src/com/company/FootballMatch.java (hunk 12 above)
Comment (AristarkhV) [reply src/com/company/FootballMatch.java:8]: Sofiia, I have some problems with github (sorry :) ), my PR is closed (can't fix it), so the PR is not updated. Should I create a new PR?

**Pull request merged**